src/Symfony/Component/Cache/Traits/PdoTrait.php

```diff
@@ -140,6 +140,17 @@ public function createTable()
     /**
      * {@inheritdoc}
      */
+    public function prune()
+    {
+        $deletes = $this->getConnection()->prepare("DELETE FROM $this->table WHERE $this->lifetimeCol + $this->timeCol <= :time");
```

Comment:
Since the `namespace` property is a `private` property of `AbstractAdapter` (well, actually the trait `AbstractTrait` but that is used by `AbstractAdapter`) how am I to determine the namespace? I think it's a fine idea, just not sure how to accomplish it.